pyvista/plotting/plotting.py

```diff
@@ -361,21 +361,37 @@ def import_gltf(self, filename, set_camera=True):
         if set_camera:
             self.camera_position = 'xy'
 
-    def export_html(self, filename):
+    def export_html(self, filename, backend='pythreejs'):
         """Export this plotter as an interactive scene to a HTML file.
 
+        You have the option of exposing the scene using either vtk.js or
+        three.js, both of which are excellent JavaScript libraries to visualize
```

Comment:
I assume `vtk.js` and `three.js` correspond to `panel` and `pythreejs`, respectively. I think we should mention this correspondence somewhere.

Reply:
```suggestion
        You have the option of exposing the scene using either vtk.js (using ``panel``) or
        three.js (using ``pythreejs``), both of which are excellent JavaScript libraries to visualize
```